packages/tools/package/package.json

```diff
@@ -30,6 +30,7 @@
     "start": "node ./index.js",
     "test": "yarn test:style && yarn test:syntax && yarn test:integration && yarn test:coverage",
     "test:coverage": "yarn test:integration --coverage --reporters=\"jest-silent-reporter\"",
+    "test:unit": "jest",
```

Comment:
Why was this needed? UTs were running for the other commands in this package before right?

Reply:
We run the `test:unit` command to run unit tests for all the packages but here we didn't have the command. Although nothing is broken because test:integration is running the unit test as you can see in the command below.

Reply:
@Shreyas281299 let's remove the test:integration command then. It's not needed.